snooty/main.py

```diff
@@ -151,7 +151,9 @@ def on_update(
         if page.query_fields:
             document.update({"query_fields": page.query_fields})
 
-        self.handle_document(build_identifiers, page_id.without_known_suffix, document)
+        self.handle_document(
+            build_identifiers, page_id, fully_qualified_pageid, document
```

Comment:
Ah, so the reason for the error was that we were trying to pass in just the `page_id`, such as `search`, but the `MongoBackend` was trying to replace the `<project>/<user>/<branch>/search` document? Is my understanding on the error correct?

Reply:
Yep, exactly. Ideally we would actually have separate types to avoid conflating fully qualified IDs with regular IDs, but anyway